### PR TITLE
istio-resources component

### PR DESCRIPTION
**Pull request by strekm**
**Description**

Changes proposed in this pull request:

- introducing new istio component

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/13113

### CODE REVIEW

**Review by mmitoraj — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

File: resources/istio-resources/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Istio Resources
+
+## Overview
+
+[Istio](https://istio.io/) is an open-source service mesh providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies, and aggregate telemetry data.
+
+The Istio Resources Helm chart consists of:
+
+- Mutual TLS (mTLS) configuration enabling mTLS cluster-wide in a STRICT mode
+- Istio monitoring configuration details
```
Comment (veichtj) [inline resources/istio-resources/README.md:10]: Should we mention that the dashboards reflect the official istio grafana dashboards?: https://istio.io/latest/docs/ops/integrations/grafana/

Comment (veichtj) [reply resources/istio-resources/README.md:10]: Agreed with @strekm will be done in follow-up PR

**Review by veichtj — APPROVED** [review]

**Review by mjakobczyk — APPROVED** [review]
Welcome brand-new Istio component 😄 💯 
/meow

**Pull request merged**